File: java/Fase2.java
package lletres_repetides;

import java.util.Arrays;
import java.util.List;

public class Fase2 {

	public static void main(String[] args) {
		
		List<Character> name = Arrays.asList('C', 'a', 'r', 'l', 'o', 's');
		
		for (Character c : name) {
			if ( c=='a' || c=='e' || c=='i' || c=='o' || c=='u' ||
					c=='A' || c=='E' || c=='I' || c=='O' || c=='U') {
				// print vowel message
				System.out.println("VOCAL");
			} else if ( c=='0' || c=='1' || c=='2' || c=='3' || c=='4' ||
					c=='5' || c=='6' || c=='7' || c=='8' || c=='9') {
				// print number message
				System.out.println("Els noms de persones no contenen números!");
			} else {
				// print consonant message
				System.out.println("CONSONTANT");
			}
		}
	}
}

File: javaScript/fase2.js
var name = ['C', 'a', 'r', 'l', 'o', 's'];

document.write("Nom: " + name + "<br><br>");

for (var i=0; i<name.length; i++) {
	if ( name[i]=='a' || name[i]=='e' || name[i]=='i' || name[i]=='o' || name[i]=='u' ||
			name[i]=='A' || name[i]=='E' || name[i]=='I' || name[i]=='O' || name[i]=='U') {
		// print vowel message
		document.write("VOCAL<br>");
	} else if ( name[i]=='0' || name[i]=='1' || name[i]=='2' || name[i]=='3' || name[i]=='4' ||
			name[i]=='5' || name[i]=='6' || name[i]=='7' || name[i]=='8' || name[i]=='9') {
		// print number message
		document.write("Els noms de persones no contenen números!<br>");
	} else {
		// print consonant message
		document.write("CONSONTANT<br>");
	}	
}	


